doc-content/jbpm-docs/src/main/asciidoc/BPMN2/appe-service-tasks.adoc

```diff
@@ -55,10 +55,10 @@ In addition to primitive object types, the WebService task can use custom object
 
 To use custom objects:
 
-. Create a custom model object using either the Data Modeler in Business Central, or using an external tool, like Red Hat JBoss Developer Studio.
+. Create a custom model object using either the Data Modeler in Business Central, or using an external tool, like Red Hat CodeReady Studio.
```

Comment:
Very minor suggestion: Instead of Business Central, just use **{CENTRAL}** attribute.

Reply:
Thanks for catching that